content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
+Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number. 
+Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
+R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174]. 
+R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 
+A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model. 
+In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively. 
+In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174]. 
 A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
-The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
+The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions were put into place.
+For example, if only a fraction S~t~ of the population is still susceptible, R~t~ = S~t~ x R~0~.
 When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
 R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
-Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
+Modeling approaches are typically based upon a classic epidemiological model structure: the SIR model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0]. 
+In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used.[need citations]
```

Comment:
```suggestion
In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used; for example, some models incorporate  population age-structure and contact patterns among different age groups [@doi:10/ggp3xq] or spatial aspects and population movement [@doi:10.1073/pnas.2004978117], and in general possible infectious disease model complexities include various heterogeneities, different biological scales, temporal forcing, spatial structure, and stochasticity [@isbn:9780199209996].
```

Reply:
@LucyMcGowan @alavendelm This was to give some citations & examples of possible model complexities. I used both a couple articles I read as examples since the sentence mentioned SARS-CoV-2 and a general ~textbook/reader in the field to cover the various possibilities; if you were thinking of other aspects or saw different types of models in the articles you read then you can adjust the sentence if needed. I wasn't sure exactly how to approach this myself because there are roles for models of different levels of complexities and sometimes the types of models that get used changes or increases over time. Whatever you guys thinks is good enough :-)

Reply:
@LucyMcGowan this looks like the only suggestion that hasn't been incorporated yet. I saw your 👍 emoji so figured you wanted to commit it but wanted to make sure.